TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8261,6 +8262,8 @@ execStatementEnd: END_EXEC;
 
 //FOR SQL
 commitStatement: SQL_COMMIT;
+rollBackStatement: SQL_ROLLBACK ({ string.Equals(CurrentToken.Text, "WORK", System.StringComparison.OrdinalIgnoreCase) }? KeywordWORK=UserDefinedWord)? savePointClause?;
```

Comment:
Since WORK is contextual but can be used at several places in the grammar, you can define it in its on rule:
`work: ({ string.Equals(CurrentToken.Text, "WORK", System.StringComparison.OrdinalIgnoreCase) }? KeywordWORK=UserDefinedWord);` and then use `work` in ROLLBACK or COMMIT